packages/project-management-automation/tsconfig.json

```
@@ -2,8 +2,19 @@
 	"$schema": "https://json.schemastore.org/tsconfig.json",
 	"extends": "../../tsconfig.base.json",
 	"compilerOptions": {
-		"rootDir": "lib",
-		"types": [ "node" ]
+		"target": "es2020",
+		"module": "commonjs",
+		"lib": [ "es2020" ],
+		"outDir": "./build",
+		"rootDir": "./lib",
+		"types": [ "node" ],
+		"checkJs": false,
+		"emitDeclarationOnly": false,
+		"composite": true,
+		"declaration": true,
+		"declarationMap": false,
+		"declarationDir": "./build-types"
 	},
-	"include": [ "lib" ]
+	"include": [ "lib/**/*" ],
```

No need to change this

```suggestion
	"include": [ "lib" ],
```